### DISCUSSION TITLE
Bug: syslog daemon changes break its usage on macOS 10.13 and above

### DISCUSSION THREAD

**Discussion by sambacha** [usnistgov/macos_security#327] [General]
### Summary

On macOS 12.x (Monterey), Apple has changed the behavior of their syslog daemon - it no longer listens on a domain socket. Therefore, you cannot reliably expect this to work on macOS 10.13 and above.

Suggestions for using syslog should be adjusted accordingly, depending on the version of macOS. 

### Possible fixes

Apple has unified the system logging and syslog no longer contains the entries we would need, even if it did work as expected. Checkout this tool https://eclecticlight.co/consolation-t2m2-and-log-utilities/  and this blog post https://eclecticlight.co/2020/02/07/how-long-does-your-mac-keep-its-log-for/ for more information.

**Comment by golbiga** [comment]
This does not appear to be an issue with the project, moving to discussions.
